pipelines/crawler/sfb_sicar/tasks.py

```
@@ -1,75 +1,171 @@
+        "MALLOC_ARENA_MAX": "2",
+        "MALLOC_TRIM_THRESHOLD_": "131072",
+    }
+    work = tempfile.mkdtemp(prefix="sicar_clean_")
+    total_rows = 0
+    total_dropped = 0
+    peak_mb = 0.0
+    try:
+        parts = extract_theme_zip(zip_path, work)
+        ranges = plan_shp_ranges(parts, budget_bytes=CLEAN_BUDGET_BYTES)
+        for idx, (shp, start, count) in enumerate(ranges):
+            proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)